os/fs/smartfs/smartfs.h

```diff
@@ -249,6 +249,8 @@
 #undef  CONFIG_SMARTFS_DYNAMIC_HEADER
 #endif
 
+#define FS_AVAIL_BYTES fs->fs_llformat.availbytes	/* Bytes in the sector excluding MTD header */
```

Comment:
The `fs` is the local variable name so that it could be changed. How about getting it from the macro?
```
#define FS_AVAIL_BYTES(fs) (fs)->fs_llformat.availbytes   /* .... */
```

Reply:
Updated